nunaliit2-couch-user/src/main/java/ca/carleton/gcrc/couch/user/mail/UserRegistrationGenerator.java

```diff
@@ -0,0 +1,38 @@
+package ca.carleton.gcrc.couch.user.mail;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Map;
+
+import ca.carleton.gcrc.mail.MailMessage;
+import ca.carleton.gcrc.mail.messageGenerator.MailMessageGenerator;
+
+class UserRegistrationGenerator implements MailMessageGenerator {
+
+    public UserRegistrationGenerator(){
+
+	}
+
+	@Override
+	public void generateMessage(
+			MailMessage message,
+			Map<String,String> parameters
+			) throws Exception {
+
+		String link = parameters.get("link");
+		String userEmail = parameters.get("userEmail");
+
+		// Subject
+		message.setSubject("Nunaliit User Registration");
+
+		// Create HTML body part
+		StringWriter sw = new StringWriter();
+		PrintWriter pw = new PrintWriter(sw);
+		pw.println("<html><head><title>Nunaliit User Registration</title></head><body><h1>Nunaliit User Registration</h1>");
+		pw.println("<p>A new user <b>" +userEmail +"</b> has been created for Nunaliit Atlas. ");

```

Comment:
`A new user <name> has registered with the atlas.`
  
Might be more consistent wording given the subject and title talked about registration as well.